README.md

```diff
@@ -3,8 +3,36 @@
 The Rancher Desktop guest agent runs in Rancher Desktop VMs providing helper
 services.  It currently has the following functionality:
 
+# When Privileged Service is enabled (admin installation of Rancher Desktop):
```

Comment:
This should be a lower level (it's less important than the document title). This means bumping the subheadings even lower.

Also, please rephrase so that it's actually a heading; maybe something like:

```suggestion
## Port forwarding with privileged helper:
When the Rancher Desktop Privileged Service is enabled on the host Windows machine via an admin installation of Rancher Desktop, the guest agent emits…
```